src/app/login/login-form.component.ts

```diff
@@ -186,8 +186,8 @@ export class LoginFormComponent {
     const data = {
       'user': 'org.couchdb.user:' + userName,
       'message': `New member <b>${addedMember}</b> has joined.`,
-      'link': '/manager/users/',
-      'linkParams': { 'search': addedMember },
+      'link': '/manager/users/profile/' + addedMember,
+      //'linkParams': { 'search': addedMember },

```

Comment:
@singharpita This line should be removed.

It also fails the lint test.  Is there an issue with `npm run install-hooks`?